hbp_knowledge/receptors/wu2011.bel

```diff
@@ -137,60 +161,74 @@ maintenance of neurites and synapses [18–20], long-term
 potentiation [95,96], seizures [97], and neuronal viability/death
 [21–24]. These intriguing ﬁndings underscore the need for further
 characterization of functional a7-nAChRs."
+SET Confidence = "High"
 a(HBP:"alpha-7-containing nAChR") -- bp(GO:"neurotransmitter secretion")
 a(HBP:"alpha-7-containing nAChR") -- bp(GO:"maintenance of synapse structure")
 a(HBP:"alpha-7-containing nAChR") -- bp(GO:"neuron projection maintenance")
 a(HBP:"alpha-7-containing nAChR") -- bp(GO:"long-term synaptic potentiation")
 a(HBP:"alpha-7-containing nAChR") -- path(MESH:Seizures)
 a(HBP:"alpha-7-containing nAChR") -- bp(GO:"neuron death")
+UNSET Confidence
 
 SET Evidence = " The fact that oligomeric amyloid-b
 seems to have high afﬁnity for a7b2-nAChRs expressed in the brain
 region that is damaged early in Alzheimer’s disease may have
 implications for disease etiopathogenesis."
+SET Confidence = "High"
 SET MeSHDisease = "Alzheimer Disease"
 SET MeSHAnatomy = "Brain"
 complex(a(HBP:"alpha-7 beta-2 nAChR"),a(CHEBI:"amyloid-beta"))
 UNSET MeSHAnatomy
 UNSET MeSHDisease
+UNSET Confidence
 
 SET Evidence = "a6 subunits are not widely expressed in the brain, but they are
 prevalent in midbrain dopaminergic (DAergic) regions associated
 with pleasure, reward, and mood control "
+SET Confidence = "High"
 SET Cell = "midbrain dopaminergic neuron"
 a(HBP:"alpha-6-containing nAChR") -- path(MESH:Pleasure)
 a(HBP:"alpha-6-containing nAChR") -- path(MESH:Reward)
 a(HBP:"alpha-6-containing nAChR") -- path(MESH:Affect)
 UNSET Cell
+UNSET Confidence
 
 SET Evidence = " Recently, we reported a novel discovery that
 functional a6*-nAChRs are located on GABAergic presynaptic
 boutons associated with VTA DAergic neurons, where these a6*-
 nAChRs mediate nicotinic modulation of GABA release onto those
 DAergic neurons [122]."
+SET Confidence = "High"
 SET MeSHAnatomy =  {"Presynaptic Terminals","GABAergic Neurons"}
 a(HBP:"alpha-6-containing nAChR") reg sec(a(CHEBI:"gamma-aminobutyric acid"))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Initially, a-CtxMII was
 thought to be highly selective for a3b2*-nAChRs [124] and became
 a useful tool for investigating receptor structure [125]."
-a(MESH:"alpha-conotoxin MII") -| act(a(HBP:"alpha-3 beta-2 nAChR"))
+SET Confidence = "Medium"
+a(MESH:"alpha-conotoxin MII") -> act(a(HBP:"alpha-3 beta-2 nAChR"))
```

Comment:
a-CtxMII is a potent and selective antagonist for a3b2*-nAChRs, I think I looked it up, because the evidence was not clear here. But it is definitively inhibiting the receptor
```suggestion
a(MESH:"alpha-conotoxin MII") -| act(a(HBP:"alpha-3 beta-2 nAChR"))
```